server/commands/hubs.py

```diff
@@ -112,7 +112,10 @@ def ooc_cmd_save_hub(client, arg):
             raise ArgumentError("Filename must be at least 3 symbols long!")
     try:
         if args[0] != "":
-            path = "storage/hubs"
+            if len(args) > 2 and args[2].lower() == "read_only":
```

Comment:
`/save_hub args[0] args[1] args[2]`, args[2] doesn't exist.
this should be `len(args) > 1 and args[1].lower()`, please check all instances of this